common-react/@dbeaver/react-data-grid/src/renderers/HeaderCellContentRenderer.tsx

```diff
@@ -1,13 +1,37 @@
-import { use } from 'react';
+import { use, useRef } from 'react';
 import { DataGridCellHeaderContext } from '../DataGridHeaderCellContext.js';
 import { useGridReactiveValue } from '../useGridReactiveValue.js';
+import { OrderButton } from './OrderButton.js';
 
 interface Props {
   colIdx: number;
+  tabIndex?: number;
 }
-export function HeaderCellContentRenderer({ colIdx }: Props) {
+export function HeaderCellContentRenderer({ colIdx, tabIndex }: Props) {
+  const attributePosition = colIdx - 1;
```

Comment:
Please remove, you need only `colIdx`, the issue that initially forced you to add this can be fixed at the Grid usage level